src/http-api/fetch.ts

```diff
@@ -225,6 +226,9 @@ export class FetchHttpApi<O extends IHttpOpts> {
         body?: Body,
         opts: Pick<IRequestOpts, "headers" | "json" | "localTimeoutMs" | "keepAlive" | "abortSignal"> = {},
     ): Promise<ResponseType<T, O>> {
+        const urlForLogs = this.clearUrlParamsForLogs(url);
+        logger.debug(`FetchHttpApi: ${method} --> ${urlForLogs}`);
```

Comment:
It's weird to separate the method from the URL like this

```suggestion
        logger.debug(`FetchHttpApi: --> ${method} ${urlForLogs}`);
```